samples/matter/common/src/persistent_storage/Kconfig

```diff
@@ -16,6 +16,10 @@ config NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND
 	select TRUSTED_STORAGE
 	select PSA_PROTECTED_STORAGE
 
+config NCS_SAMPLE_MATTER_SECURE_STORAGE_MAX_KEY_LEN
```

Comment:
Could we change the name, so it would not suggest it applies only to secure storage?

Reply:
Ehhh, sorry, sure.